media_driver/linux/common/ddi/media_libva_common.cpp

```diff
@@ -352,3 +353,86 @@ void* DdiMedia_GetContextFromVABufferID (PDDI_MEDIA_CONTEXT mediaCtx, VABufferID
 
     return ctx;
 }
+
+int32_t DdiMedia_GetGpuPriority(
+    VADriverContextP     ctx,
+    VABufferID           *buffers,
+    int32_t              numBuffers,
+    bool                 *bPriorityUpdate,
```

Comment:
Please follow/align the naming rules of variables.

Reply:
Done.